.github/workflows/run_examples_polynomial.yml

```diff
@@ -86,7 +87,8 @@ jobs:
       env:
         ERT_STORAGE_DATABASE_URL: postgresql://postgres:postgres@localhost:${{ job.services.postgres.ports[5432] }}/postgres
       run: |
-        ert-storage &
+        ert api &
```

Comment:
I presume `ert api` is responsible for starting the instance and `ert3 service` for checking (and waiting for the status to be up)? If so, perhaps something like `ert3 service check --all` (or `ert3 service check storage`) would be a more natural start that will allow us to extend the cli later on? Breaking changes is of course not a big problem here, but I still see the tendency that early implementation is used as firm hints for the future 🤷🏻 Or, alternatively leave a few reflecting comments in the existing "Service" issue for the future?

Reply:
I agree, we should have a discussion for how the services arguments in ert3 should be set up. I didnt want to impose much design at the moment, as I had not made too much a reflection as to what would be best.

Reply:
I agree that we should minimise the design impact until we do the design. However, I vote for doing `ert3 service check storage` and let that be the only valid combination for now. Then it is at least clear what one is doing...

Reply:
Done!